gubernator/kubelet_parser.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import datetime
+import os
+import re
+
+import jinja2
+
+def parse(lines, error_re, hilight_res, filters):
+    matched_lines = []
+    UID = ""
+
+    end = 0
+    event_re = re.compile(r'.*Event\(api\.ObjectReference.*UID:&#34;(.*)&#34;, A.*')
```

Comment:
- nit: Do you need to encode the `"` as `&#34;` here? If it needs to be escaped, would `\"` work instead?
- Do you need the `A` after the UID pattern? I'm nervous about depending on the field ordering
- Do you need the `.*` before `Event`?
- It might be good to match the close of the object reference, so you don't accidentally match a UID outside of it. Something like:
```
Event\(api\.ObjectReference\{[^}]*UID:&#34;(.*)&#34;(, [^}]*)?\}
```